### ISSUE TITLE
Nach verdursten weiter laufen

### ISSUE THREAD

**Issue by bizzel-CJ** [AlternateLife/Bugs#187]
### Beschreibe den Fehler
Sobald man verdurstet geht man zu Boden allerdings ohne die Animation. Man kann weiter Laufen/Rennen und springen.

### Reproduktion des Fehlers
1. Verdursten

### Erwartetes Verhalten
Die "Du bist am Boden" Animation

### Name der/des beteiligten Charakter(e)
Chris Jordan

### System
- Betriebssystem: Windows 10
- Browser: Chrome
- GTA Version: steam

### Screenshots
http://prntscr.com/olzvy2

**Comment by Cryma** [comment]
Ich kann das nicht reproduzieren. Ist dir das schon mehrmals passiert oder bisher nur einmal?

**Comment by bizzel-CJ** [comment]
Nein, bisher erst einmal danach bin ich "leider" nicht mehr verdurstet. Vielleicht zur erklärung: Ich stand am Hotelzimmer habe mich nicht bewegt kurze Zeit später stand bei mir dann "Du bist am Boden" aber stattdessen die Animation Kriechen kommt konnte ich ganz normal weiter laufen oder rennen. Vielleicht war ich auch raus getabt und dadurch kam es zu einem Char Freeze was erklären würde warum ich keine Animation hatte aber da ich es nicht mehr genau weiß kann ich es auch nicht genau sagen. Sorry
